### PR TITLE
NAS-103897 / 12.0 / Nas 103897

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/pages/sharing/sharing.routing.ts [hunk 1]
```diff
@@ -98,6 +99,10 @@ export const routes: Routes = [
         path: 'edit/:pk',
         component: SMBFormComponent,
         data: { title: 'Edit', breadcrumb: 'Edit' },
+      },{
+        path: 'acl/:pk',
+        component: SMBAclComponent,
+        data: { title: 'Share ACL', breadcrumb: 'Share ACL'},
```
Comment (erinix) [inline src/app/pages/sharing/sharing.routing.ts]: we should put a T() on this one

**Comment by erinix** [comment]
could you please provide some tooltips @freenas/docs-team

File: src/app/helptext/sharing/smb/smb.ts [hunk 2]
```diff
@@ -122,5 +126,28 @@ export const helptext_sharing_smb = {
...
+    ae_who_sid_tooltip: T(''),
+
+    ae_who_name_domain_placeholder: T('Domain'),
+    ae_who_name_domain_tooltip: T(''),
+
+    ae_who_name_name_placeholder: T('Name'),
+    ae_who_name_name_tooltip: T(''),
+
+    ae_perm_placeholder: T('Permission'),
+    ae_perm_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/sharing/smb/smb.ts]:
```
Predefined permission combinations:<br>
<i>Read</i>: Read access and Execute permission on the object (RX).<br>
<i>Change</i>: Read access, Execute permission, Write access, and Delete object (RXWD).<br>
<i>Full</i>: Read access, Execute permission, Write access, Delete object, change Permissions, and take Ownership (RXWDPO).<br><br>
For more details, see <a href="https://www.samba.org/samba/docs/current/man-html/smbcacls.1.html" target="_blank">smbacls(1)</a>.
```

File: src/app/helptext/sharing/smb/smb.ts [hunk 3]
```diff
@@ -122,5 +126,28 @@ export const helptext_sharing_smb = {
...
+
+    ae_type_placeholder: T('Type'),
+    ae_type_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/sharing/smb/smb.ts]: How permissions are applied to the share. <i>Allowed</i> denies all permissions by default except those that are manually defined. <i>Denied</i> allows all permissions by default except those that are manually defined.

File: src/app/helptext/sharing/smb/smb.ts [hunk 4]
```diff
@@ -122,5 +126,28 @@ export const helptext_sharing_smb = {
...
+
+    // share acl
+    share_acl_basic: T('Basic'),
+    share_acl_entries: T('ACL Entries'),
+
+    share_name_placeholder: T('Share Name'),
+    share_name_tooltip: T(''),
+
+    ae_who_sid_placeholder: T('SID'),
+    ae_who_sid_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/sharing/smb/smb.ts]:
```
Who this ACL entry applies to, shown as a
<a href="https://docs.microsoft.com/en-us/windows/win32/secauthz/security-identifiers" target="_blank">Windows Security Identifier</a>.
Either a <i>SID</i> or a <i>Domain</i> and <i>Name</i> is required for this ACL.
```

File: src/app/helptext/sharing/smb/smb.ts [hunk 5]
```diff
@@ -122,5 +126,28 @@ export const helptext_sharing_smb = {
...
+
+    ae_who_name_domain_placeholder: T('Domain'),
+    ae_who_name_domain_tooltip: T(''),
+
+    ae_who_name_name_placeholder: T('Name'),
+    ae_who_name_name_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/sharing/smb/smb.ts]:
```
Who this ACL entry applies to, shown as a user name. Requires adding the user <i>Domain</i>.
```

File: src/app/helptext/sharing/smb/smb.ts [hunk 6]
```diff
@@ -122,5 +126,28 @@ export const helptext_sharing_smb = {
...
+    share_acl_entries: T('ACL Entries'),
+
+    share_name_placeholder: T('Share Name'),
+    share_name_tooltip: T(''),
+
+    ae_who_sid_placeholder: T('SID'),
+    ae_who_sid_tooltip: T(''),
+
+    ae_who_name_domain_placeholder: T('Domain'),
+    ae_who_name_domain_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/sharing/smb/smb.ts]:
```
Domain for the user <i>Name</i>. Required when a <i>SID</i> is not entered.
Local users have the SMB server NetBIOS name: <code>freenas\\smbusers</code>.
```

**Pull request merged**